### ISSUE TITLE
swiftlint disable warnings

### ISSUE THREAD

**Issue by Igor-Palaguta** [sunshinejr/Pouch#1]
Hello @sunshinejr 

Nice tool!

Please can you add to template swiftlint disabling, e.g. for my swiftlint configuration can see 3 warnings:

```
/Users/igorpalaguta/Projects/BookFusion/BookFusion/BFDownloadManager/DownloadManagerSecrets.swift:1:1: File Header Violation: Header comments should be consistent with project patterns. (file_header)
/Users/igorpalaguta/Projects/BookFusion/BookFusion/BFDownloadManager/DownloadManagerSecrets.swift:10:13: Implicit Return Violation: Prefer implicit returns in closures, functions and getters. (implicit_return)
/Users/igorpalaguta/Projects/BookFusion/BookFusion/BFDownloadManager/DownloadManagerSecrets.swift:6:1: Line Length Violation: Line should be 200 characters or less: currently 2165 characters (line_length)

```

As an example you can check what SwiftGen disables https://github.com/SwiftGen/SwiftGen/blob/stable/templates/strings/structured-swift5.stencil

**Comment by sunshinejr** [comment]
hey @Igor-Palaguta - thanks for this, I think will be a very nice addition. Do you mind playing with the PR yourself? I think this should be as simple as putting the `disable all` flag at [SwiftGenerator.swift#L34-L42](https://github.com/sunshinejr/Pouch/blob/main/Sources/PouchFramework/Generators/Swift/SwiftGenerator.swift#L34-L42). To test if it works, you can clone the repo and use `make uninstall/install` to test the local version of the tool (also remove any brew-installed versions).

**Comment by Igor-Palaguta** [comment]
Yes sure, let me prepare PR, what do you also think about switching from 3 to default 4 spaces?

**Comment by sunshinejr** [comment]
@Igor-Palaguta ah yes, it might be a bug if there is 3-spaces-indentation somewhere, so a fix for that would be also great to have.

**Comment by Igor-Palaguta** [comment]
Correspondent PR https://github.com/sunshinejr/Pouch/pull/2

**Comment by sunshinejr** [comment]
@Igor-Palaguta I just merged the PR & released version 0.2.0 both on GH & on my brew tap - thanks again! 🙌
